clients/pkg/promtail/targets/syslog/syslogtarget.go

```diff
@@ -1,8 +1,11 @@
 package syslog
 
 import (
-	"errors"
 	"fmt"
+	"github.com/efficientgo/core/errors"
+	"golang.org/x/text/encoding"
+	"golang.org/x/text/encoding/ianaindex"
+	"golang.org/x/text/transform"
```

Comment:
Please move into 2nd import block. The first block is for stdlib imports, the 2nd one for external libraries, and the 3rd one for project internal imports.